enclave_wrangler/objects_api.py

```diff
@@ -324,41 +324,77 @@ def update_db_with_new_objects(objects=None):
     pass
 
 
-def get_new_objects(since: datetime = None):
-    """Get new objects"""
-    # https://www.palantir.com/docs/foundry/api/ontology-resources/objects/object-basics/#filtering-objects
-    # https://unite.nih.gov/workspace/data-integration/restricted-view/preview/ri.gps.main.view.af03f7d1-958a-4303-81ac-519cfdc1dfb3
-    yesterday = (datetime.now() - timedelta(
-        days=1)).isoformat() + 'Z'  # works: 2023-01-01T00:00:00.000Z
-    since = since if since else yesterday
+# TODO: finish this
+def get_all_new_objects(since: Union[datetime, str]) -> Dict[str, List]:
```

Comment:
`get_all_new_objects()`: New func that calls and uses `get_new_cset_and_member_objects()` and also gets researchers, projects, etc (WIP)